hack/setup-e2e-env.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -e
+
+echo "Create k3d cluster"
+# kubernetes < 1.25 is currently required by rancher
+k3d cluster create -i "rancher/k3s:v1.24.10-k3s1" kw-policy-e2e
+
+echo "Add cert-manager helm chart repository"
+helm repo add jetstack https://charts.jetstack.io
+
+echo "Add rancher-latest helm chart repository"
+helm repo add rancher-latest https://releases.rancher.com/server-charts/latest
+
+echo "Update local Helm chart repository cache"
+helm repo update
+
+echo "Install cert-manager CRDs"
+kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.7.1/cert-manager.crds.yaml
+
+echo "Install the cert-manager Helm chart"
+helm install cert-manager jetstack/cert-manager \
```

Comment:
```suggestion
helm install --wait cert-manager jetstack/cert-manager \
```

Even if we have a timeout and wait for the Project CRD in Rancher, this adds robustness.

Reply:
No longer around, we are not testing in full isolation mode using a new feature of kwctl